build_and_push.sh

```diff
@@ -59,9 +61,16 @@ function build_and_push_manifest() {
     done
 }
 
+function read_config() {
+    CONFIG_KEY="$1"
```

Comment:
this should probably also be a local variable. No need to expose it to the global namespace